test/cli/run.spec.ts

```diff
@@ -611,6 +613,43 @@ Options:
     });
   });
 
+  describe("handles ast output", () => {
+    it("conflicts with --test/--test-file", async() => {
+      await exec({
+        args: ["--ast", "--test", '1'],
+        stdin: 'foo = "1"',
+        exitCode: 1,
+        expected: "CommanderError: error: option '-a, --ast' cannot be used with option '-t, --test <text>'",
+      });
+      await exec({
+        args: ["--ast", "--test-file", 'file'],
+        stdin: 'foo = "1"',
+        exitCode: 1,
+        expected: "CommanderError: error: option '-a, --ast' cannot be used with option '-t, --test <text>'",
+      });
+    });
+
+    it("produces AST", async() => {
+      const output = await exec({
+        args: ["--ast"],
+        stdin: 'foo = "1"',
+        exitCode: 0,
+      });
```

Comment:
For some unknown reason for me this stucks and failed after timeout, although manual run with
```console
echo "foo = '1'" | node .\bin\peggy.js --ast
```
worked as expected. I ran out of ideas, need help. Maybe it is related to that fact that here `outputString` is always `null` at line 188:
https://github.com/peggyjs/peggy/blob/46878cd11debdc1f9b45f6705f00f74f38c19cd6/test/cli/run.spec.ts#L188-L196

Reply:
Found the issue. The AST string is bigger than 16k, the default highwaterMark for the MockStream, which means that the write blocks.  The quick fix is to add: `others.highWaterMark = 32768;` at line 35 in the `MockStream` constructor before the `super` call.  I'll work on a better fix in a later patch.

Reply:
Ah.. I can wait until you properly fix that

Reply:
I'll fix it in #287.